try:
    x = 1
except ValueError:
    x = 2
finally:
    x = 3
